src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -104,91 +104,65 @@ public ExamResource(UserRepository userRepository, CourseRepository courseReposi
     }
 
     /**
-     * POST /courses/{courseId}/exams : Create a new exam.
+     * POST courses/{courseId}/exams : Create a new exam.
      *
      * @param courseId  the course to which the exam belongs
      * @param exam      the exam to create
      * @return the ResponseEntity with status 201 (Created) and with body the new exam, or with status 400 (Bad Request) if the exam has already an ID
-     * @throws URISyntaxException if the Location URI syntax is incorrect
      */
-    @PostMapping("/courses/{courseId}/exams")
+    @PostMapping("courses/{courseId}/exams")
```

Comment:
just for ℹ️
Is removing preceding slash a new pattern we have in place? Asking due to [this](https://github.com/ls1intum/Artemis/pull/3910#discussion_r694791886).

Reply:
I answered to the discussion there, sorry I missed it for so long ^^' To summarize, yes it's been part of the server coding guidelines for about 2 months and I am working on removing them all.